include/RigidBody/Joints.h

```diff
@@ -163,6 +163,28 @@ class BIORBD_API Joints : public RigidBodyDynamics::Model
     int GetBodyRbdlIdToBiorbdId(
         const int idx) const;
 
+    ///
+    /// \brief Return the rbdl idx of child of each segments
+    /// \return the rbdl idx of child of each segments
+    ///
+    std::vector<std::vector<unsigned int> > GetChildRbdlIdx() const;
+
+    ///
+    /// \brief Return the rbdl idx of subtrees of each segments
+    /// \return the rbdl idx of subtrees of each segments
+    ///
+    std::vector<std::vector<unsigned int> > GetSubTrees();
```

Comment:
SubTrees is way too generic. When you choose a function name, think of it as if someone were to give you a code with the minimal amount of lines, as such: 
```python
import biorbd
m = biorbd.Model('model.bioMod')
print(m.GetSubTrees())
```
would you be able to tell what it prints?

Reply:
It prints the subtrees for each dofs, i.e. all the descendants.

Reply:
DofSubTrees ?

Reply:
Modify with getDofSubTrees